### ISSUE TITLE
Clear Combox

### ISSUE THREAD

**Issue by Fruchtgummi** [andlabs/ui#204]
How i do clear the Combobox?

**Comment by andlabs** [comment]
What do you mean by clearing it?

**Comment by andlabs** [comment]
What do you mean by clearing it? Removing entries or clearing the editable part?

**Comment by Fruchtgummi** [comment]
Yes, removing entries! And what about tables?

**Comment by ernierasta** [comment]
Hi!
Maybe it is not best solution, but it works for me.

Lets say we have:
``` go
box := ui.NewHorizontalBox()
c := ui.NewCombobox()
c.Append("foo")
box.Append(c, false)
```
Recreate combo with different options:
``` go
box.Delete(0) // delete Combobox from parent box
c.Destroy() // destroy Combobox
c := ui.NewCombobox() // recreate empty
c.Append("bar") // add new options
box.Append(c, false) // re-append it to box
```
Tested only on Linux.

**Comment by andlabs** [comment]
Merged with andlabs/libui#309.

(I will be making a master issue for things missing from package ui later.
